src/main/java/io/qdrant/client/QdrantClient.java

```diff
@@ -665,6 +674,76 @@ public ListenableFuture<List<AliasDescription>> listAliasesAsync(@Nullable Durat
 
 	//endregion
 
+	//region ShardKey Management
+
+		/**
+	 * Creates a shard key for a collection.
+	 *
+	 * @param createShardKey The request object for the operation.
+	 * @return a new instance of {@link ListenableFuture}
+	 */
+	public ListenableFuture<CreateShardKeyResponse> createShardKeyAsync(CreateShardKeyRequest createShardKey) {
+		return createShardKeyAsync(createShardKey, null);
+	}
+
+	/**
+	 * Creates a shard key for a collection.
+	 *
+	 * @param createShardKey The request object for the operation.
+	 * @param timeout The timeout for the call.
+	 * @return a new instance of {@link ListenableFuture}
+	 */
+	public ListenableFuture<CreateShardKeyResponse> createShardKeyAsync(CreateShardKeyRequest createShardKey, @Nullable Duration timeout) {
+		String collectionName = createShardKey.getCollectionName();
+		Preconditions.checkArgument(!collectionName.isEmpty(), "Collection name must not be empty");
+		logger.debug("Create shard key'{}'", collectionName);
```

Comment:
log both shard key and collection name

```suggestion
		logger.debug("Create shard key '{}' for '{}'", shardKey, collectionName);
```